src/com/amazon/corretto/crypto/provider/EvpEdEcKey.java

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider;
+
+abstract class EvpEdEcKey extends EvpKey {
```

Comment:
Drop the EC, this naming convention further increases the confusion around all things related to elliptic curves.

Reply:
Updated!